kazoo/tests/test_threading_handler.py

```diff
@@ -45,10 +45,6 @@ def test_double_start_stop(self):
         assert h._running is False
 
     def test_huge_file_descriptor(self):
-        from kazoo.handlers.threading import _HAS_EPOLL
```

Comment:
I don't think this test has any remaining value with your change. This is really the responsibility of the various `selectors` module.

But I would like to see some unit tests for the   "`select()` API -> `selectors()` API -> `select()` result" conversions.

Reply:
I just get the `select` test case code from official python test suite to test the `selector_select` and cover the normal `select` usage. See the file `kazoo/tests/test_selectors_select.py`